mod/edit/set/all/form_buttons.rb

```diff
@@ -3,6 +3,18 @@ def standard_submit_button
     output [standard_save_button, standard_save_and_close_button]
   end
 
+  # Generates a standard save button with optional customization.
+  # @param opts (dict, optional): A dictionary containing customization options
+  #   - If voo is not nil and does not hide the save button, a save button is generated
+  #   - If present, "text" key in opts sets the text content of the button
+  #   - "data-cy" attribute is set to "save" for end-to-end testing purposes
+  #   - CSS class "submit-button me-3 _update-history-pills" is applied
+  # Returns:
+  #   - None: If the save button is not generated due to the hide condition.
+  # Example:
+  #   standard_save_button({"text": "Submit", "data-cy": "submit_button"})
+  #   This example generates a customized save button with the text "Submit"
```

Comment:
you wouldn't ever set "data-cy" to anything; it will just get overwritten. `:text`, on the other hand, used `||=`, so you can set that as an option. All other options are passed on to `#submit_button`